src/components/TaskList/TaskList.jsx

```diff
@@ -85,12 +85,15 @@ class TaskList extends React.Component {
   refresh(event) {
     event.currentTarget.blur() // defocus button
     const github = new GitHub()
-    github.getTasks(this.props.activeFilter.query)
-      .then(result => {
-        const tasks = result.tasks
-        const filter = this.props.activeFilter
-        this.props.dispatch({ type: 'TASKS_UPDATE', filter, tasks })
-      })
+    github.getTasks(this.props.activeFilter).then(result => {
+      const tasks = result.tasks
+
+      // GitHub's api doesn't like when the output includes milliseconds
+      const updatedAt = (new Date()).toISOString().replace(/\.\d{3}Z/, 'Z')
```

Comment:
Haha considering your comment, I bet there's a story here about how you discovered this particular limitation.